utils/data-migration/gef/migrate-deal-files.js

```diff
@@ -0,0 +1,67 @@
+const {
```

Comment:
Do we need this file? We have in portal-api the migration script for files :)
(not a show stopper - we can do it after the merge)